code4me-server/markdowns/index.md

```diff
@@ -1,25 +1,28 @@
 # Code4Me
-**Code4Me** is a plugin that autocompletes code in a TU Delft research project for the CSE3000 course given in the Computer Science and Engineering bachelor.
-This study focuses on the usefulness of various statement completion language models in a real-world setting.
+Code4Me provides language model based code completion. Code4Me predicts statement (line) completion and is available for both Jetbrains and Visual Studio Code. The code suggestions from Code4Me can be recognised by the logo in the suggestion menu. Code4Me triggers (on specific trigger characters) automatically or it can be prompted manually by pressing the keybind. This keybind differs per IDE.
 
-The plugin autocompletes on trigger characters, and a keybind which will trigger the autocompletion wherever you want.
+Code4Me does not hinder native auto completion or other extensions.
 
 ## Installation
-We have developed IDE plugins for the following two major IDEs:
+The plugin is available on both marketplaces. You can download them by either clicking the link in your browser or looking up 'Code4Me' in the integrated marketplace of your IDE.
 
 ### JetBrains
 The Code4Me plugin can be found in the [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/19200-code4me).
 
 ### VSC
-TODO
+The Code4Me plugin can be found in the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=Code4Me.code4me-plugin).
 
 ## Data Collection
-The plugin works with a remote API, which requires us to send (parts) of the currently opened file to our backend server, in order to suggest autocompletions.
-However, we do not store any of this information server-side, it is purely used for generating an autocompletion.
-After this request the server will return a prediction to the client.
-The prediction made by the model (the completion of the line) is stored on the server.
-In the plugin we track the position of the completion which allows us to send the ground truth (possibly changed version of the prediction) back to the server after 30 seconds.
-The ground truth is stored and compared to the initial prediction using different evaluation metrics to evaluate the autocompletion model.
+The plugin works with a remote API in order to suggest autocompletions. After this request the server will return a prediction to the client. The remote API requires a segment of the 2048 characters. Thus this limited segment is sent to the TU Delft backend server. We do not store the entire segment. The inserted line is tracked over the course of 30 seconds and then returned to the server and then stored. This line has to be stored to use different evaluation metrics to properly asses the autocompletion model.
+
+The plugin does not collect personal data nor the full segment sent. The plugin does collect the following data:
```

Comment:
```suggestion
The plugin does not collect personal data nor the segment sent. The plugin does collect the following data:
```

Reply:
I think this is a tricky one since we could store a part of the segment if someone removes the insertion...

Reply:
If someone removes the insertion, it will not store anything from the segment tho?

Reply:
Ye, unless that line is directly edited, but I guess it's fine.